packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2214,6 +2215,9 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 return { stage: "base", referenceSequenceNumber: summaryRefSeqNum, error };
             }
             const { summary: summaryTree, stats: partialStats } = summarizeResult;
+            const gcSummaryTreeStats = summaryTree.tree.gc
```

Comment:
Please don't use ".gc" directly as it assumes the GC tree's key is gc. Use `gcTreeKey` which is defined in `GarbageCollection` class.